packages/office-addin-dev-settings/test/integration-tests/test.ts

```diff
@@ -25,7 +23,9 @@ if (process.platform === "win32") { // only windows is supported
       sandbox.restore();
     });
     it("loopback already enabled", async function() {
+      const command: commander.Command = new commander.Command();
```

Comment:
The issue here is really that loopback isn't already enabled on the test machine. The test never enables it originally, and because it is not enabled it is showing the prompt. This fixes the test to enable it without the prompt, but doesn't really configure setup or cleanup to save the current state, reset the state so it is not enabled, then have a test case to enable it.